recipes/cath-tools/build.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+set -x -e
+
+export INCLUDE_PATH="${PREFIX}/include"
+export LIBRARY_PATH="${PREFIX}/lib"
+export LD_LIBRARY_PATH="${PREFIX}/lib"
+export CFLAGS="-I$PREFIX/include"
+export CPATH="${PREFIX}/include"
+export LDFLAGS="-L${PREFIX}/lib"
+export CXXFLAGS="-I${PREFIX}/include"
+
+
+#list programs
+CATH_PROGRAMS="build-test cath-assign-domains cath-cluster cath-map-clusters cath-refine-align cath-resolve-hits cath-score-align cath-ssap cath-superpose"
```

Comment:
Is this environment variable needed while running CMake? If not, then move it down to a line after the `make`.

Reply:
Yes not needed for compilation, but many recipes do like that, I thought it was a standard to have this list on the top of the file to quickly understand what tools will be installed.
I moved it following your advise